test_util/test_upgrade_vpc.py

```diff
@@ -463,6 +590,9 @@ def run_test(self) -> int:
             self.log.info("Test failed! VPC cluster will remain available for "
                           "debugging for 2 hour after instantiation.")
 
+        if os.getenv('CI_FLAGS'):
+            result = 0
+
```

Comment:
This bit shouldn't be here, looks like it came from https://github.com/dcos/dcos/pull/1373.